qiskit_experiments/framework/experiment_data.py

```diff
@@ -107,6 +107,15 @@ def parse_utc_datetime(dt_str: str) -> datetime:
     return dt_utc
 
 
+def get_job_status(job: Job) -> JobStatus:
+    """Get job status in JobStatus format which defined in:
+    `from qiskit.providers.jobstatus import JobStatus`"""
+    status = job.status()
+    if isinstance(status, str):
+        status = getattr(JobStatus, status, JobStatus.DONE)
```

Comment:
How do you weigh defaulting to `DONE` here versus raising an error? All strings expected from qiskit-ibm-runtime have attributes on `JobStatus` so to need the default would mean that we are getting some unexpected value from qiskit-ibm-runtime.